packages/paste-core/components/breadcrumb/stories/index.stories.tsx

```diff
@@ -49,3 +51,81 @@ export const AnchorsWithTruncation = (): React.ReactNode => {
 AnchorsWithTruncation.story = {
   name: 'Anchors with truncation ',
 };
+
+export const CustomizationBreadcrumb = (): React.ReactNode => {
+  const currentTheme = useTheme();
+  return (
+    <CustomizationProvider
+      theme={currentTheme}
+      elements={{
+        BREADCRUMB: {fontVariantNumeric: 'slashed-zero'},
+        BREADCRUMB_ITEM: {fontWeight: 'fontWeightMedium'},
+        BREADCRUMB_ITEM_ANCHOR: {
+          textDecoration: 'underline wavy',
+          color: 'colorTextInverseWeaker',
+          ':hover': {color: 'colorLinkStronger'},
+        },
+        BREADCRUMB_ITEM_TEXT: {letterSpacing: '0.25rem'},
+        BREADCRUMB_SEPARATOR: {
+          color: 'colorTextBrandHighlight',
+        },
```

Comment:
Are we open to the following public API?

```
<nav data-paste-element="BREADCRUMB">
  <ol data-paste-element="BREADCRUMB_LIST">
    <li data-paste-element="BREADCRUMB_ITEM">
      <a href="" data-paste-element="BREADCRUMB_ANCHOR" />
      <span  data-paste-element="BREADCRUMB_SEPARATOR" />
    </li>
    <li data-paste-element="BREADCRUMB_ITEM">
      <a href="" data-paste-element="BREADCRUMB_ANCHOR" />
      <span  data-paste-element="BREADCRUMB_SEPARATOR" />
    </li>
    <li data-paste-element="BREADCRUMB_ITEM">
      <span data-paste-element="BREADCRUMB_TEXT" />
    </li>
  </ol>
</nav>
```

Then work backwards to achieve it?

Reply:
@SiTaggart I included `Item` in the anchor and text element names respectfully because they are included currently in the `BreadcrumbItem` component. Are you saying that we should not apply that convention rigidly here?

Reply:
FWIW I think I'm getting thrown a bit here because:
1. the component name is `BreadcrumbItem` so I am using `BREADCRUMB_ITEM` as the default element name, and passing it to all nodes rendered by the component (that we want to target)
2. It makes sense to me that the `li` is the only node that has `_ITEM` in the name, because it is literally an item.

Reply:
We don't need to expose our internal implementation details in a public API, yeah. Design for the customer experience.